lib/std/fmt/parse_float.zig

```diff
@@ -34,7 +34,7 @@
 // - Only supports round-to-zero
 // - Does not handle denormals
 
-const std = @import("../std.zig");
+const std = @import("std");
```

Comment:
Why was this changed? I think ../std.zig is more correct.

Reply:
There is a pretty much even divide between `@import("std.zig")` and `@import("std")`, I changed this because I wanted to test this file specifically.

Reply:
Generally, std lib should import std.zig by file path, setting an example for other packages. For `zig test` you can use `--main-pkg-path` to test files that import with "..".